AWSAuthSDK/Sources/AWSMobileClient/AWSUserPoolOperationsHandler.swift

```diff
@@ -82,13 +81,14 @@ AWSCognitoUserPoolInternalDelegate {
             return AWSCognitoIdentityUserPool.default()
         }
         let clientKey = "CognitoUserPoolKey"
-        guard let client = AWSCognitoIdentityUserPool.init(forKey: clientKey) else {
+        let client = AWSCognitoIdentityUserPool.init(forKey: clientKey)
```

Comment:
maybe simplify this to `if let userPoolClient = client { return client }`

Reply:
Cannot do because it will end in compiler warning.